src/main/java/core/basesyntax/service/operations/OperationType.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.operations;
+
+public enum OperationType {
+    b, s, p, r

```

Comment:
It's better to use full name for enum and assign to each of them string value b, s, p...